### PR TITLE
Docs: Fix double word typos

### PR DESCRIPTION
**Pull request by Sumit112192**
Fixed a grammar typo in the docs of kivy lang.

Maintainer merge checklist
* [x] Title is descriptive/clear for inclusion in release notes.
* [x] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [x] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [ ] Properly documented, including `versionadded`, `versionchanged` as needed.

### CODE REVIEW

**Comment by Sumit112192** [comment]
I have made another commit where I have fixed double-word typos in all the doc strings of .py files.

File: kivy/core/camera/__init__.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def _update(self, dt):
         pass
 
     def _copy_to_gpu(self):
-        '''Copy the the buffer into the texture'''
+        '''Copy the buffer into the texture'''
```
Comment (pythonic64) [inline kivy/core/camera/__init__.py:121]: @Sumit112192  Add "." at the end of the sentence.

Comment (Sumit112192) [reply kivy/core/camera/__init__.py]: Added

**Review by misl6 — APPROVED** [review]
LGTM. Thank you!

And thank you @pythonic64 for the review!

**Pull request merged**